src/Phinx/Console/Command/Rollback.php

```diff
@@ -61,6 +62,10 @@ protected function configure()
 If you have a breakpoint set, then you can rollback to target 0 and the rollbacks will stop at the breakpoint.
 <info>phinx rollback -e development -t 0 </info>
 
+The <info>version_order</info> configuration option is used to determine the order of the migrations when rolling back.
+This can be used to allow rolling back the last executed migration instead of the last created one, or combined with the
```

Comment:
wouldn't that need a `the` before, as in: `to allow ___the___ rolling back ___of___ the last executed migration`?